rotkehlchen/chain/evm/decoding/morpho/utils.py

```diff
@@ -106,6 +108,34 @@ def query_morpho_vaults(database: 'DBHandler') -> None:
     )
 
 
+def query_morpho_reward_distributors() -> None:
+    """Query Morpho reward distributor addresses from the api and cache them."""
+    try:
+        response_data = requests.get(
+            url=MORPHO_REWARDS_API,
+            timeout=CachedSettings().get_timeout_tuple(),
+        )
+        distributors: dict[ChainID, set] = {}
+        for item in response_data.json()['data']:
+            chain_id = ChainID.deserialize(item['chain_id'])
+            if chain_id not in distributors:
+                distributors[chain_id] = set()
+
+            distributors[chain_id].add(deserialize_evm_address(item['distributor']['address']))
+    except (requests.RequestException, JSONDecodeError, TypeError, KeyError) as e:
```

Comment:
`requests.RequestException` handles `JSONDecodeError` so there's no need for that.
also, what can raise `TypeError` here?